### DISCUSSION TITLE
[RFC] Improve progress reporting (Completed/In-Progress/Remaining)

### DISCUSSION THREAD

**Discussion by sppatel** [vercel/turborepo#7262] [Ideas]
### Describe the feature you'd like to request

When running task(s) across a large number of packages, it would be super helpful to be able to get a bit more insight as to how many tasks have completed/running/remaining.  

### Describe the solution you'd like

I could imagine when turbo is dumping its console output - each line could be prefixed with (6/10/20) where the first number represents successfully completed tasks, the second number how many currently in progress, and the last number how many remaining.

### Describe alternatives you've considered

No real good options to get this level of progress.

**Comment by mehulkar** [comment]
This seems useful!

**Comment by anthonyshew** [comment]
There is a new terminal UI in 2.0 that helps along with this. Let us know if it still leaves more to be desired.
